test/dev-router.unit.js

```diff
@@ -147,12 +147,34 @@ test('[dev-router] match with needed prefixed slash', async t => {
     dest: '/some/dest',
     userDest: true,
     status: undefined,
-    headers: undefined,
+    headers: {},
     uri_args: {},
     matched_route: {
       src: '^\\/([^\\/]+?)\\/comments(?:\\/)?$',
       dest: '/some/dest'
     },
     matched_route_idx: 0
   })
-})
\ No newline at end of file
+})
+
+test('[dev-router] `continue: true`', async t => {
+  const routesConfig = [
+    {
+      src: '/_next/static/(?:[^/]+/pages|chunks|runtime)/.+',
+      continue: true,
+      headers: {
+        'cache-control': 'immutable,max-age=31536000'
+      }
+    }
+  ];
```

Comment:
It seems like your test should include two routes to prove that `continue` is working.